scripts/exregional_process_radarref.sh

```diff
@@ -205,8 +207,8 @@ if [ -s filelist_mrms ]; then
    print_info_msg "$VERBOSE" "Using radar data from: `head -1 filelist_mrms | cut -c10-15`"
    print_info_msg "$VERBOSE" "NSSL grib2 file levels = $numgrib2"
 else
-   echo "ERROR: Not enough radar reflectivity files available."
-   exit 1
+   echo "WARNING: Not enough radar reflectivity files available for loop ${bigmin}."
+   continue
```

Comment:
For data process, we only give warning. The application that use those data will be skipped if there are no data. But no hard exit for those data process. Need to avoid Non-Zero exit as much as possible.